src/components/Tabs/Settings/VectorFieldSettings.tsx

```diff
@@ -49,18 +48,6 @@ export default function VectorFieldSettings(): ReactElement {
 
   return (
     <>
-      <SettingsItem label={"Show vector arrows"} htmlFor={VectorSettingsHtmlIds.SHOW_VECTOR_ARROWS_CHECKBOX}>
-        <div style={{ width: "fit-content" }}>
-          {/* TODO: Replace with a top-level checkbox for Vector arrows when Collapse menus are removed */}
-          <Checkbox
-            id={VectorSettingsHtmlIds.SHOW_VECTOR_ARROWS_CHECKBOX}
-            checked={vectorVisible}
-            onChange={(e) => setVectorVisible(e.target.checked)}
-            disabled={dataset === null}
-          />
-        </div>
-      </SettingsItem>
-
```

Comment:
<img width="628" height="341" alt="Image" src="https://github.com/user-attachments/assets/849ea039-9d12-4257-841e-dad0e9a5979e" />

This is now a higher-level checkbox that controls whether this component (VectorFieldSettings) is shown or not.